From b696d98c55d289518c8af9692291a9453a54df20 Mon Sep 17 00:00:00 2001
From: "F. von Gellhorn" <flinux@vongellhorn.ch>
Date: Tue, 31 Dec 2019 15:48:47 +0100
Subject: [PATCH 3/3] Solus tweaks to the default config

- Remove toolbox icon from desktop
- Decrease desktop icon sizes
- Set default Monospace font size to 10
- Move showdesktop in front of the systemtray
---
 containments/desktop/package/contents/config/main.xml         | 4 ++--
 kcms/fonts/fonts.cpp                                          | 2 +-
 .../org.kde.plasma.desktop.defaultPanel/contents/layout.js    | 3 ++-
 3 files changed, 5 insertions(+), 4 deletions(-)

diff --git a/containments/desktop/package/contents/config/main.xml b/containments/desktop/package/contents/config/main.xml
index 43a4935..78fb7ac 100644
--- a/containments/desktop/package/contents/config/main.xml
+++ b/containments/desktop/package/contents/config/main.xml
@@ -100,7 +100,7 @@
     </entry>
     <entry name="iconSize" type="Int">
       <label>The icon size to use for Folder View icons. One of the list of Plasma icon sizes.</label>
-      <default>4</default>
+      <default>3</default>
     </entry>
     <entry name="labelWidth" type="Int">
       <label>The width of the grid cells when using icon view. 0 = Narrow, 1 = Medium, 2 = Wide</label>
@@ -138,7 +138,7 @@
     </entry>
     <entry name="showToolbox" type="Bool">
       <label>Whether to show the Desktop Toolbox. Only used when serving as containment.</label>
-      <default>true</default>
+      <default>false</default>
     </entry>
     <entry name="pressToMove" type="Bool">
       <label>Whether to activate widget handles and widget move mode by long press. Only used when serving as containment.</label>
diff --git a/kcms/fonts/fonts.cpp b/kcms/fonts/fonts.cpp
index 3aa146f..652bc72 100644
--- a/kcms/fonts/fonts.cpp
+++ b/kcms/fonts/fonts.cpp
@@ -506,7 +506,7 @@ void KFonts::defaults()
 #else
     setGeneralFont(QFont("Noto Sans", 10));
     setMenuFont(QFont("Noto Sans", 10));
-    setFixedWidthFont(QFont("Hack", 9));
+    setFixedWidthFont(QFont("Hack", 10));
     setToolbarFont(QFont("Noto Sans", 10));
     setSmallFont(QFont("Noto Sans", 8));
     setWindowTitleFont(QFont("Noto Sans", 10));
diff --git a/layout-templates/org.kde.plasma.desktop.defaultPanel/contents/layout.js b/layout-templates/org.kde.plasma.desktop.defaultPanel/contents/layout.js
index 048671e..c17410d 100644
--- a/layout-templates/org.kde.plasma.desktop.defaultPanel/contents/layout.js
+++ b/layout-templates/org.kde.plasma.desktop.defaultPanel/contents/layout.js
@@ -78,7 +78,8 @@ if (langIds.indexOf(languageId) != -1) {
     panel.addWidget("org.kde.plasma.kimpanel");
 }
 
+panel.addWidget("org.kde.plasma.showdesktop")
 panel.addWidget("org.kde.plasma.systemtray")
 panel.addWidget("org.kde.plasma.digitalclock")
-panel.addWidget("org.kde.plasma.showdesktop")
+
 
-- 
2.24.1

